packages/trader/src/AppV2/Utils/contract-description-utils.tsx

```diff
@@ -14,123 +14,145 @@ type TVideoVariants = {
     light: string;
 };
 
-export const TERM = {
-    BARRIER: localize('Barrier'),
-    CONTRACT_VALUE: localize('Contract value'),
-    DEAL_CANCELLATION: localize('Deal cancellation'),
-    ENTRY_SPOT: localize('Entry spot'),
-    EXIT_SPOT: localize('Exit spot'),
-    EXPIRY: localize('Expiry'),
-    FINAL_PRICE: localize('Final price'),
-    GROWTH_RATE: localize('Growth rate'),
-    PAYOUT: localize('Payout'),
-    PAYOUT_PER_POINT: localize('Payout per point'),
-    PREVIOUS_SPOT_PRICE: localize('Previous spot price'),
-    RANGE: localize('Range'),
-    SLIPPAGE_RISK: localize('Slippage risk'),
-    STOP_OUT_LEVEL: localize('Stop out level'),
-    STOP_LOSS: localize('Stop loss'),
-    STRIKE_PRICE: localize('Strike price'),
-    TAKE_PROFIT: localize('Take profit'),
-};
+export const getTerm = () => ({
+    BARRIER: { value: 'Barrier', text: <Localize i18n_default_text='Barrier' /> },
+    CONTRACT_VALUE: { value: 'Contract value', text: <Localize i18n_default_text='Contract value' /> },
+    DEAL_CANCELLATION: { value: 'Deal cancellation', text: <Localize i18n_default_text='Deal cancellation' /> },
+    ENTRY_SPOT: { value: 'Entry spot', text: <Localize i18n_default_text='Entry spot' /> },
+    EXIT_SPOT: { value: 'Exit spot', text: <Localize i18n_default_text='Exit spot' /> },
+    EXPIRY: { value: 'Expiry', text: <Localize i18n_default_text='Expiry' /> },
+    FINAL_PRICE: { value: 'Final price', text: <Localize i18n_default_text='Final price' /> },
+    GROWTH_RATE: { value: 'Growth rate', text: <Localize i18n_default_text='Growth rate' /> },
+    PAYOUT: { value: 'Payout', text: <Localize i18n_default_text='Payout' /> },
+    PAYOUT_PER_POINT: { value: 'Payout per point', text: <Localize i18n_default_text='Payout per point' /> },
+    PREVIOUS_SPOT_PRICE: { value: 'Previous spot price', text: <Localize i18n_default_text='Previous spot price' /> },
+    RANGE: { value: 'Range', text: <Localize i18n_default_text='Range' /> },
+    SLIPPAGE_RISK: { value: 'Slippage risk', text: <Localize i18n_default_text='Slippage risk' /> },
+    STOP_OUT_LEVEL: { value: 'Stop out level', text: <Localize i18n_default_text='Stop out level' /> },
+    STOP_LOSS: { value: 'Stop loss', text: <Localize i18n_default_text='Stop loss' /> },
+    STRIKE_PRICE: { value: 'Strike price', text: <Localize i18n_default_text='Strike price' /> },
+    TAKE_PROFIT: { value: 'Take profit', text: <Localize i18n_default_text='Take profit' /> },
```

Comment:
Can we keep the same TERM object (constant) and use its keys as values? we would only replace small localize to big Localize. 
We could do this in `GuideDefinitionModal`:
```
{TERM[term]?.text} // instead of {term?.text}
```
and this:
```
{getTermDefinition({ term, contract_type })} // instead of {getTermDefinition({ term: term?.value, contract_type })}
```
We just meed to set `term` as a key instead of an object in trade types descriptions like `VanillasTradeDescription`: 
```
onClick={() => onTermClick('PAYOUT')} // instead of onClick={() => onTermClick(PAYOUT)}
```
willl it work? :)

Reply:
I tried with object.keys(), but the order is a bit random, so I reworked my solution by adding localize (string) which can be used as key